packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -36,6 +36,8 @@ export class RemoteMessageProcessor {
 	 * @remarks For chunked batches, this is the CSN of the "representative" chunk (the final chunk)
 	 */
 	private batchStartCsn: number | undefined;
+	private readonly processorBatch: InboundSequencedContainerRuntimeMessageOrSystemMessage[] =
```

Comment:
Future: We should be able to just skip calling into RemoteMessageProcessor altogether if it's not a ContainerRuntime message (according to `modernRuntimeMessage` flag in calling function)

Reply:
Actually we may just want to do this now, looking at the rest of the change.